### DISCUSSION TITLE
QuickStart link in the cloud portal 404s

### DISCUSSION THREAD

**Discussion by SilentRhetoric** [WalletConnect/walletconnect-monorepo#4043] [Developer Support] [answered]
The Web3Modal Quick Start link inside the project settings tab of the cloud platform is a dead link:

https://docs.walletconnect.com/quickstart?utm_source=cloud-app&utm_campaign=quickstart

**Comment by glitch-txs** [comment, marked as answer]
Thanks for reporting:
https://docs.walletconnect.com/getting-started
